instrumentation/jaxrs-client/jaxrs-client-2.0-testing/build.gradle.kts

```diff
@@ -24,7 +25,8 @@ dependencies {
   latestDepTestLibrary("org.glassfish.jersey.inject:jersey-hk2:2.+")
   latestDepTestLibrary("org.glassfish.jersey.core:jersey-client:2.+")
   latestDepTestLibrary("org.jboss.resteasy:resteasy-client:3.0.26.Final")
-  latestDepTestLibrary("org.apache.cxf:cxf-rt-rs-client:3.+")
+  // latestDepTestLibrary("org.apache.cxf:cxf-rt-rs-client:3.+")
```

Comment:
do you think it would be worth to create an issue and add reference here as a comment to provide more context and deal with that later ?

Reply:
I was thinking of just creating an issue. There is a good chance that this can be reverted in a couple of days.